be/src/http/action/restore_tablet_action.cpp

```diff
@@ -121,36 +121,27 @@ Status RestoreTabletAction::_reload_tablet(
                      << ", signature: " << tablet_id;
         // remove tablet data path in data path
         // path: /roo_path/data/shard/tablet_id
-        std::string tablet_path = shard_path + "/" + std::to_string(tablet_id);
+        std::string tablet_path = shard_path + "/" + std::to_string(tablet_id) + "/" + std::to_string(schema_hash);
```

Comment:
You can use strings::Substitute to finish it

Reply:
ok